doc/source/news.rst

```diff
@@ -5,6 +5,50 @@
 News
 ====
 
+.. _release-11-0-6:
+
+Release 11.0.6 - 2021-08-29
+---------------------------
+
+Improvements
+------------
+
+* Added support for recovering on crash. (experimental)
+
+  * This is a experimental feature. Currently, this feature is still not stable.
+
+  * If Groonga crashes, it recovers the database automatically when it open a database for the first time since the crash. However, This feature can't recover the database automatically in all crash cases.
+    We need to recover the database manually depending on timing even if this feature enable.
+
+* [:doc:`reference/executables/grndb`] Added ``dump-wal`` command.
+
+* Added a new tool ``dump-wal.rb`` for dumping WAL.
+
+* 
+
+Fixes
+-----
+
+* Fixed a bug that Groonga doesn't clear lock when some threads open the same object around the same time.
+
+  If some threads open the same object around the same time, threads except for a thread that executes the opening object at first are waiting for opening the target object.
+  At this time, threads that are waited for an opening object take locks, but these locks are not released.
```

Comment:
Present tense: threads that **wait** for an opening object take locks,

Reply:
Thanks! I've fixed it.